cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -91,10 +128,60 @@ def issue_jwt_token_not_memoized_key(token_body, algorithm = Algorithms::RS256)
     )
   end
 
+  def issue_jwt_ecdsa_token(token_body, algorithm = Algorithms::ES256)
+    @jwt_token = JWT.encode(
+      token_body,
+      ecdsa_private_key,
+      algorithm,
+      {
+        kid: jwk_ecdsa.kid
+      }
+    )
+  end
+
+  def issue_jwt_hmac_token(token_body, algorithm = Algorithms::HS256)
+    @jwt_token = JWT.encode(
+      token_body,
+      HMAC_SECRET,
+      algorithm
+    )
+  end
+
+  def issue_rsa_jwt_token_with_alg_header(token_body, alg_header, algorithm = Algorithms::RS256)
+    encoded_token = JWT.encode(
+      token_body,
+      rsa_keys[@default_file_name],
+      algorithm
+    )
+    token_parts = encoded_token.split(".")
+    headers = "{\"typ\":\"JWT\",\"alg\":\"#{alg_header}\",\"kid\":\"#{jwk_set[@default_file_name].kid}\"}"
+    token_parts[0] = Base64.urlsafe_encode64(headers).strip
+    @jwt_token = token_parts.join(".")
+  end
+
+  def issue_jwt_hmac_token_token_with_rsa_key(token_body)
+    @jwt_token = JWT.encode(
+      token_body,
+      rsa_keys[@default_file_name].to_s,
```

Comment:
I believe this is already a string (we dont convert to string in other places that do the same thing)
```suggestion
      rsa_keys[@default_file_name],
```

Reply:
Its not string its object. HMAC should be string.